Releasenotes.md

```diff
@@ -1,6 +1,76 @@
 # Release notes BRK-Bevragen
 
-## Versie 1.1.1:
+## **Versie 1.2.0:**
+
+### openapi.yaml
+
+- Algemeen
+  - Diverse verbeteringen in de documentatie (descriptions) en examples.
```

Comment:
ik zou bovenaan zetten dat we nieuwe functionaliteit hebben toegevoegd: hypotheken, beslagen, privaatrechtelijke beperkingen en publiekrechtelijke beperkingen. Plus adressen bij kadastraalonroerendezaken en aanschrijfnaam bij kadaster natuurlijk personen.

Reply:
ik zou ook ergens ruimte maken voor open issues. Bijvoorbeeld/met name:
- #521: bij zoeken personen worden niet alle gegevens geleverd
- #725: bij zoeken natuurlijk personen worden aanschrijfwijze, aanhef en gebruikInLopendeTekst niet correct gevuld

Reply:
ook van belang: een aantal properties is depricated verklaard, omdat ze niet (meer) geleverd kunnen worden

Reply:
In mijn beleving is dit geen boek waarbij we een samenvatting aan het begin moeten zetten. Alles in de releasenotes is van belang en punten 1 en 3 zijn genoemd.

> ik zou ook ergens ruimte maken voor open issues. Bijvoorbeeld/met name:

Ik zal deze 2 issues nu noemen omdat dat opgenomen functionaliteit is die nog niet afdoende werkt. Maar verder:  Welke issues ga ik wel noemen en welke niet ?  Er staat 181 issues open.

Reply:
> In mijn beleving is dit geen boek waarbij we een samenvatting aan het begin moeten zetten. Alles in de releasenotes is van belang en punten 1 en 3 zijn genoemd.

@JohanBoer ik ben het niet met je eens. Ik denk dat iemand in een oogopslag moet zien wat de release voor mogelijk interessante nieuwe functionaliteit levert, zonder alle details door te hoeven werken. Aangezien de release geen breaking changes bevat (minor release) kan een gebruiker ervoor kiezen de details te negeren. Maar de echt nieuwe functionaliteit wel. Als ik nu een kopje Algemeen lees met alleen documentatie, exempels, gebruik externe specificatie, enz. dan krijg ik niet de indruk dat deze release vooral een hoop belangrijke nieuwe functionaliteit levert. Dus ik zou eerst de belangrijkste dingen benoemen, en de administratieve wijzigingen onderaan zetten